components/src/molecules/rate/Rate.stories.js

```diff
@@ -0,0 +1,95 @@
+import { storiesOf } from '@storybook/vue'
+import { action } from '@storybook/addon-actions'
+
+import ARate from './Rate.vue'
+
+const info = `
+  ---
+  #### Use these modifiers with \`.radio\` class.
+  - \`.rating--large\` - Selector for applying large styles
+  ---
+`
+
+storiesOf('Molecules/Rate', module)
+  .addParameters({ info })
+  // @vue/component
+  .add('Default', () => ({
+    components: { ARate },
+    data: () => ({
+      rating: 2.4,
+      ratingItems: 5
+    }),
+    template: `
+       <a-rate 
+        v-model="rating"
+        :rating-items="ratingItems"
+        disabled
+        legend="Your rating"
+       />
+    `
+  }))
+  // @vue/component
+  .add('Rate', () => ({
+    components: { ARate },
+    data: () => ({
+      rating: 1,
+      ratingItems: 4
+    }),
+    methods: {
+      select: action('Selected'),
+      ariaLabelRate () {
+        return `Rate option, ${this.rating} of ${this.ratingItems}. Click to vote`
+      }
+    },
+    template: `
+      <a-rate 
+       v-model="rating"
+       :rating-items="ratingItems"
+       :aria-label-rate="ariaLabelRate()"
+       legend="Your rating"
+       @change="select"
+      />
+    `
+  }))
+  // @vue/component
+  .add('Large', () => ({
+    components: { ARate },
+    data: () => ({
+      rating: 2.4,
+      ratingItems: 5
+    }),
+    template: `
+      <a-rate 
+       class="rating--large"
+       v-model="rating"
+       :rating-items="ratingItems"
+       disabled
+       legend="Your rating"
+      />
+    `
+  }))
+  // @vue/component
+  .add('With slot', () => ({
```

Comment:
Rename it to `Rate with slot`. I also suggest to change the order of stories to have:
1. Default
2. Large
3. Rate
4. Rate with slot